advocacy_docs/partner_docs/VeritasGuide/05-UsingNetBackupForPostgreSQL.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Using NetBackup for PostgreSQL'
+description: 'Walkthroughs of example usage scenarios'
```

Comment:
How is their admin server used?  is there any admin interface to see how things are working?  Is it only command line?

Reply:
GUI (Java-based) admin console - screenshot added in the guide